core/vm/evm.go

```diff
@@ -84,6 +84,7 @@ type BlockContext struct {
 
 	// Arbitrum information
 	ArbOSVersion uint64
+	BaseFeeCopy  *big.Int // Copy of BaseFee to be used in arbitrum's geth hooks and precompiles when BaseFee is lowered to 0 when vm runs with NoBaseFee flag and 0 gas price. Is nil when BaseFee isn't lowered to 0
```

Comment:
Realized the name is confusing.

BaseFeeInBlock?
BlockBaseFeeForEthCall? something else?

Reply:
changed it